pkg/types/baremetal/defaults/platform_test.go

```diff
@@ -144,8 +144,8 @@ func TestSetPlatformDefaults(t *testing.T) {
 				ProvisioningBridge:      "",
 				ProvisioningNetwork:     baremetal.DisabledProvisioningNetwork,
 				ProvisioningNetworkCIDR: machineNetwork,
-				APIVIP:                  "192.168.111.2",
-				IngressVIP:              "192.168.111.3",
+				APIVIPs:                 []string{"192.168.111.2"},
+				IngressVIPs:             []string{"192.168.111.3"},
 			},
```

Comment:
Can we add tests for IPv6 addresses in APIVIPs and IngressVIPs and also for a dual stack case where 2 IP addresses are provided in APIVIPs and IngressVIPs, one ipv4 and v6 address each?

Reply:
In the test for the vip validation, we have a test with both API vips being IPv4 and IPv6:

https://github.com/openshift/installer/blob/e7c39f77ce018823a05caf962dc94493aa11ad87/pkg/types/validation/installconfig_test.go#L1606-L1631

Reply:
AFAICT, these tests are all about validating the derived values for the platform. E.g. when the provisioning CIDR is set to 1.1.1.0/24 the default provisioning IPs should be 1.1.1.2 and 1.1.1.3. The only logic related to VIPs here is the optional DNS lookup, which is already being covered.

As Christoph mentioned, there are a lot of tests covering v4 and v6 in the install_config validation tests, so many in fact that GitHub won't load the diff by default. :-)

Reply:
Thanks for pointing them out. I found them in a diff commit.